test/functional/tool-data/sample_tool_data_tables.xml

```diff
@@ -11,4 +11,7 @@
     <columns>value, dbkey, name, path</columns>
     <file path="${__HERE__}/fasta_indexes.loc" />
   </table>
+  <table name="testdatabase" comment_char="#" type="database">
```

Comment:
Should this not have a comment_char?

Reply:
I do think that's not needed, I've now removed it.